src/Sylius/Bundle/UiBundle/Resources/views/_stylesheets.html.twig

```diff
@@ -0,0 +1,7 @@
+<link rel="stylesheet" href="{{ asset('css/style.css') }}">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
```

Comment:
imo font-awesome should be required via npm and published via a gulp task. 
Updating font awesome would require a `npm update && gulp` only